odiseoteam/sylius-vendor-plugin/1.2/config/packages/odiseo_sylius_vendor_plugin.yaml

```diff
@@ -0,0 +1,2 @@
+imports:
+    - { resource: "@OdiseoSyliusVendorPlugin/Resources/config/config.yaml" }
```

Comment:
In this file you keep config to `knp_gaufrette` , `vich_uploader` and `liip_imagine`. Some of that config will override the any existing config and cause issues really hard to find. 

Im not a big fan of solutions like this.

Reply:
@Nyholm We are only defining the configuration that we are going to use in this plugin, which will merge with the configuration of each project.
What solution do you think is correct?